builder/esp32.py

```diff
@@ -125,7 +125,7 @@ def save(self):
             )
 
         if not os.path.exists(f'{SCRIPT_DIR}/build'):
-            os.mkdir(f'{SCRIPT_DIR}/build')
+            os.makedirs(f'{SCRIPT_DIR}/build')
```

Comment:
this doesn't need to be changed because `SCRIPT_DIR` is where the `make.py` script is located. If that directory doesn't exist then there is a really HUGE problem. `mkdir` will create a single directory so long as all other directories under it exist. So in this case every other directory under the build directory MUST exist because they ran the `make.py` script

Reply:
I think I fixed that by https://github.com/lvgl-micropython/lvgl_micropython/pull/156/commits/7a532e85cff325707d499f8fc6f5bd2b50ebf338